### PR TITLE
Feature: Support non-pattern paragraphs in widget and formatter

### PR DESCRIPTION
**Pull request by recrit**

### CODE REVIEW

File: patternbuilder.admin.inc [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <?php
+
```
Comment (andrewmriley) [inline patternbuilder.admin.inc:2]: The blank line isn't needed here.

File: theme/theme.inc [hunk 2]
```diff
@@ -1,11 +1,12 @@
 <?php
+
```
Comment (andrewmriley) [inline theme/theme.inc:2]: Blank line not required

File: patternbuilder.admin.inc (hunk 1 above)
Comment (recrit) [inline patternbuilder.admin.inc:2]: This gets flagged by phpcs for the Drupal standard "Drupal.WhiteSpace.OpenTagNewline.BlankLine"

**Comment by andrewmriley** [comment]
Reviewed

**Pull request merged**